src/components/NewsTeaserItem/__tests__/NewsTeaserItem.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, fireEvent, getByTestId } from "@testing-library/vue";
+import NewsTeaserItem from "./../";
+
+describe("components/NewsTeaserItem", () => {
+  it("calls handleClick callback on call", async () => {
+    const title = "title test";
+    const date = "02.02.2020";
+    const description = "desc test";
+    const spy = jest.fn();
+    const { getByTestId } = render(NewsTeaserItem, {
+      props: {
+        title: title,
+        date: date,
+        description: description,
+        handleClick: spy,
+      },
+    });
+    const readMore = getByTestId("newsteaseritem-click");
+    await fireEvent(readMore!, new Event("click"));
+    expect(spy).toHaveBeenCalled();
+  });
+  it("checks if passed content renders", () => {
+    const title = "title test";
+    const date = "02.02.2020";
+    const description = "desc test";
+    const { getByTestId } = render(NewsTeaserItem, {
+      props: {
+        title: title,
+        date: date,
+        description: description,
+      },
+    });
+    const titleTest = getByTestId("newsteaseritem-title");
+    const dateTest = getByTestId("newsteaseritem-date");
+    const descTest = getByTestId("newsteaseritem-description");
+    expect(titleTest).toBeTruthy();
+    expect(dateTest).toBeTruthy();
+    expect(descTest).toBeTruthy();
+  });
+  it("checks if latest is working on true", () => {
+    const title = "title test";
+    const date = "02.02.2020";
+    const description = "desc test";
+    const { getByTestId } = render(NewsTeaserItem, {
+      props: {
+        title: title,
+        date: date,
+        description: description,
+        latest: true,
+      },
+    });
+    const latestTest = getByTestId("newsteaseritem-latest");
+    expect(latestTest!.classList.contains("is-latest")).toBeTruthy();
```

Comment:
This test does not work! I checked by replacing "is-latest" with "is-latest2" -> the test is still green. What works is "toBe(true)" instead of "toBeTruthy". I'm new to this JavaScript stuff and maybe there is reason for toBeTruthy to exist.. but until I got further explanation I'll just stick to https://vincenttunru.com/toBeTruthy-vs-toBe-true/